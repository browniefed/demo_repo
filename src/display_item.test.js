import React from 'react';
import DisplayItem from './display_item';
import renderer from 'react-test-renderer';

it('renders loading', () => {
  const tree = renderer.create(
    <DisplayItem 
      loading
    />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});


it('renders data', () => {
  const tree = renderer.create(
    <DisplayItem 
      productName="Test"
      price={10}
      description="Description"
    />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});