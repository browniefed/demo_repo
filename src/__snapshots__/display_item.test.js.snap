// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders data 1`] = `
<div
  className="display_item"
>
  <h3>
    Test
     - $
    10
  </h3>
  <p>
    Description
  </p>
</div>
`;

exports[`renders loading 1`] = `
<div>
  Loading...
</div>
`;
